### PR TITLE
Update .gitignore

### PR DESCRIPTION
**Pull request by oamsath**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/oamsath/functional-doge/pull/1" target="_blank">ReviewNinja

File: .gitignore [hunk 1]
```diff
@@ -91,4 +91,4 @@ UpgradeLog*.XML
```
Comment (oamsath) [inline .gitignore]: test

File: .gitignore [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ## Ignore Visual Studio temporary files, build results, and
```
Comment (oamsath) [inline .gitignore]: sdfsdf

File: .gitignore (hunk 1 above)
Comment (oamsath) [inline .gitignore]: !resolve

Comment (oamsath) [inline .gitignore]: !resolved

File: .gitignore [hunk 3]
```diff
@@ -91,4 +91,4 @@ UpgradeLog*.XML
 UpgradeLog*.htm
 
 # Microsoft Fakes
-FakesAssemblies/
```
Comment (oamsath) [inline .gitignore]: 2114

Comment (oamsath) [inline .gitignore]: !completed

**Pull request closed without merging**